torch/testing/_internal/common_methods_invocations.py

```diff
@@ -220,6 +221,33 @@ def sample_inputs(self, device, dtype, requires_grad=False):
                                         requires_grad=requires_grad)),)
 
 
+def fast_growing_function_wrapper(fn):
+    # NumPy promotes integer types to double while
+    # PyTorch promotes integer to float.
+    #
+    # This affects operators whose outputs grow very fast
+    # eg. sinh, exp, etc.
+    # To mitigate that, we force NumPy to compute with the
+    # dtype that Pytorch uses for those computations.
+    #
+    # Code Example
+    # >>> x = torch.tensor(501.)
+    # >>> x.exp()
+    # tensor(inf)
+    # >>> x = torch.tensor(501., dtype=torch.double)
+    # >>> x.exp()
+    # tensor(3.8154e+217, dtype=torch.float64)
+    def is_integral(dtype):
+        return dtype in [np.uint8, np.int8, np.int16, np.int32, np.int64]
```

Comment:
+ np.bool?

Reply:
Ah right. 
`bool` is unlikely to have the same issue.
But will add it for consistency. 

Thanks!